src/lib/import-export/export/exporters/default-exporter.ts

```diff
@@ -120,10 +121,8 @@ export class DefaultExporter extends EventEmitter implements Exporter {
 	private createArchive(): archiver.Archiver {
 		this.emit( ExportEvents.BACKUP_CREATE_START );
 		const isZip = this.options.backupFile.endsWith( '.zip' );
-		return archiver( isZip ? 'zip' : 'tar', {
-			gzip: ! isZip,
-			gzipOptions: isZip ? undefined : { level: 9 },
-		} );
+		const format = isZip ? 'zip' : 'tar';
+		return archiver( format, ARCHIVER_OPTIONS[ format ] );
```

Comment:
Before the change, if a user chose the `.zip` file, Studio passed gzip options in the form of `{ gzip: false, gzipOptions: undefined }`. It didn't do a harm as it used non-zero default compression for the ZIP file internally, but it's better to handle options in both cases explicitly.